.travis.yml

```diff
@@ -2,4 +2,30 @@ language: java
 sudo: false
```

Comment:
Thanks for the extra explanation, that helped me!  And we can defer the automagic handling 'til later.